utils/utils.py

```diff
@@ -668,41 +523,22 @@ def run_benchmarks(
         Specify the memory of your gpu(This controls the lines to be used. Also work for CPU version).
     extended_functionality : bool, default: False
         Extends functionality of H2O benchmark by adding 'chk' functions and verbose local reporting of results.
-    db_server : str, default: "localhost"
-        Host name of MySQL server.
-    db_port : int, default: 3306
-        Port number of MySQL server.
-    db_user : str, optional
-        Username to use to connect to MySQL database. If user name is specified, script attempts to store
-        results in MySQL database using other `db-*` parameters.
-    db_pass : str, default: "omniscidb"
-        Password to use to connect to MySQL database.
-    db_name : str, default: "omniscidb"
-        MySQL database to use to store benchmark results.
-    db_table_etl : str, optional
-        Table to store ETL results for this benchmark.
-    db_table_ml : str, optional
-        Table to store ML results for this benchmark.
+    db_config: DbConfig
+        Configuration for the database
     commit_hdk : str, default: "1234567890123456789012345678901234567890"
         HDK commit hash used for benchmark.
     commit_omniscripts : str, default: "1234567890123456789012345678901234567890"
         Omniscripts commit hash used for benchmark.
     commit_modin : str, default: "1234567890123456789012345678901234567890"
         Modin commit hash used for benchmark.
     """
-    ignore_fields_for_results_unit_conversion = [
-        "Backend",
-        "dfiles_num",
-        "dataset_size",
-        "query_name",
-    ]
 
     # Set current backend, !!!needs to be run before benchmark import!!!
     set_backend(pandas_mode=pandas_mode, ray_tmpdir=ray_tmpdir, ray_memory=ray_memory)
 
-    run_benchmark = __import__(benchmark_mapper[bench_name]).run_benchmark
+    benchmark: BaseBenchmark = __import__(benchmark_mapper[bench_name]).Benchmark()
```

Comment:
It could potentially be like this:
```suggestion
    benchmark: BaseBenchmark = Benchmark(__import__(benchmark_mapper[bench_name]).run_benchmark)
```

Reply:
Ok

Reply:
I think current version is better

Reply:
If, for instance, provides interface for `run_benchmark` function, instead of just expecting any function

Reply:
It allows allows us to potentially let benchmarks provide more data for running scripts than we currently have. For example, we could add new method for Benchmark and let individual benchmarks change that method to specify behavior.

Reply:
Makes sense.